src/config.js

```diff
@@ -633,7 +633,15 @@ export default class Config {
       const ws = extractWorkspaces(manifest);
       if (ws && ws.packages) {
         const relativePath = path.relative(current, initial);
-        if (relativePath === '' || micromatch([relativePath], ws.packages).length > 0) {
+        // Make packages trailing-slash-agnostic
+        const sanitizedPackages = ws.packages.map(packageGlob => {
+          if (packageGlob.endsWith('/')) {
+            packageGlob = packageGlob.slice(0, -1);
+          }
+          return packageGlob + '{,/}';
```

Comment:
Why not just replace this function by `return packageGlob.replace(/\/?$/, '')`? It would ensure that no pattern would be ended with a trailing slash, which seems what we want?